docker/Dockerfile.release

```
@@ -0,0 +1,76 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2020-2021 NVIDIA CORPORATION
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+# -------------------------------------------------- #
+# This is a Docker image presents out-of-source
+# build. If you want a release build of tritonserver
+# with dali_backend and don't want to build whole
+# tritonserver, start from here.
+# -------------------------------------------------- #
+
+ARG BASE_IMAGE=nvcr.io/nvidia/tritonserver:21.04-py3
+FROM ${BASE_IMAGE} as builder
+
+RUN apt-get update && \
+    apt-get install -y software-properties-common
+
+RUN add-apt-repository ppa:deadsnakes/ppa && \
+    apt-get update && \
+    apt-get install -y  \
+              zip              \
+              wget             \
+              build-essential  \
+              autoconf         \
+              autogen          \
+              unzip            \
+              python3.8        \
+              python3-pip      \
+              libboost-all-dev \
+              rapidjson-dev
+
```

```suggestion
RUN apt-get update && \
    apt-get install -y software-properties-common && \
    add-apt-repository ppa:deadsnakes/ppa && \
    apt-get update && \
    apt-get install -y  \
              zip              \
              wget             \
              build-essential  \
              autoconf         \
              autogen          \
              unzip            \
              python3.8        \
              python3-pip      \
              libboost-all-dev \
              rapidjson-dev && \
    rm -rf /var/lib/apt/lists/*;
```
I would do this in one go and clean up.